timesys/high-level-triggers.py

```diff
@@ -601,7 +607,8 @@
         },
     'BO-Glob:TI-Mags-Corrs': {
         'database': {
-            'Src': {'value': 0, 'enums': ('RmpBO', 'OrbBO', 'Cycle', 'Study')},
+            'Src': {'value': 0, 'enums': (
+                'Cycle', 'Study', 'Linac', 'RmpBO')},
```

Comment:
Could we keep 'RmpBO' as fisrt option here?

Reply:
@anacso17, I decided to remove RmpBO, Linac and Study from this file, since they will be available for all triggers. However, the order of the events will change, because I'll add these three events after the ones defined here. Do you see a problem with this approach?

Reply:
The suggestion was just to try to keep the most used sources in the same index, but I think it's more interesting to keep the same pattern for all triggers, changing the index of all of them